netty/src/main/java/io/grpc/transport/netty/GrpcSslContexts.java

```diff
@@ -43,6 +47,16 @@
 public class GrpcSslContexts {
   private GrpcSslContexts() {}
 
+  private static ApplicationProtocolConfig DEFAULT_APN = new ApplicationProtocolConfig(
+          Protocol.ALPN,
+          SelectorFailureBehavior.FATAL_ALERT,
```

Comment:
FYI: https://github.com/netty/netty/pull/3767

I don't know what that means we want to do. I don't know what actual behavior we want.